gateway-js/src/index.ts

```diff
@@ -203,10 +203,11 @@ export class ApolloGateway implements GraphQLService {
   private updateServiceDefinitions: Experimental_UpdateComposition;
   // how often service defs should be loaded/updated (in ms)
   private experimental_pollInterval?: number;
-  // Configure the endpoint by which gateway will access its precomposed schema.
-  // * `string` means use that endpoint
+  // Configure the endpoints by which gateway will access its precomposed schema.
+  // * An array of URLs means use these endpoints to obtain schema, if one is unavailable then try the next.
   // * `undefined` means the gateway is not using managed federation
-  private schemaConfigDeliveryEndpoint?: string;
+  private schemaConfigDeliveryEndpoints?: string[];
+  private schemaDeliveryMaxRetries?: number;
```

Comment:
Can we rename these to match the other rename we did? `uplinkXYZ`?

Reply:
done